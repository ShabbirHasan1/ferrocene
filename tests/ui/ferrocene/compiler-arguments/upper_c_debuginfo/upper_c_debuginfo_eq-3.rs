// check-fail
// compile-flags: -Cdebuginfo=3

fn main() {}

// ferrocene-annotations: um_rustc_C_debuginfo
