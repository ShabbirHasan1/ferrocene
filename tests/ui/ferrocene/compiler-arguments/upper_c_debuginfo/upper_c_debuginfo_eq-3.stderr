error: incorrect value `3` for codegen option `debuginfo` - either an integer (0, 1, 2), `none`, `line-directives-only`, `line-tables-only`, `limited`, or `full` was expected

